dask/dataframe/core.py

```diff
@@ -1087,8 +1087,8 @@ class Series(_Frame):
     _name: str
         The key prefix that specifies which keys in the dask comprise this
         particular Series
-    name: scalar or None
-        Series name.  This metadata aids usability
+    meta: pd.Series
+        An empty Series with appropriate name and dtype.
```

Comment:
define appropriate.  Perhaps "An empty Pandas Series with name and type matching the expected output" (if that fits)